pkg/util/install/utils.go

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package install
+
+import (
+	"fmt"
+	"strings"
+
+	iotv1alpha1 "github.com/enmasseproject/enmasse/pkg/apis/iot/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Apply some default deployment values
+func ApplyDeploymentDefaults(deployment *appsv1.Deployment, component string, app string, name string) {
+	if deployment.Labels == nil {
+		deployment.Labels = make(map[string]string)
+	}
+
+	deployment.Labels["enmasse-component"] = component
+	deployment.Labels["app"] = app
+	deployment.Labels["name"] = name
+
+	deployment.Spec.Selector = &v1.LabelSelector{
+		MatchLabels: map[string]string{
+			"enmasse-component": component,
+			"app":               app,
+			"name":              name,
+		},
+	}
+
+	if deployment.Spec.Template.ObjectMeta.Labels == nil {
+		deployment.Spec.Template.ObjectMeta.Labels = make(map[string]string)
+	}
+
+	deployment.Spec.Template.ObjectMeta.Labels["enmasse-component"] = component
```

Comment:
I think just 'component' is sufficient. If you have other things using that label, you can further distinguish using app=enmasse.

Reply:
True. Changed and pushed.